cmd/netpolicy/cmd/evaluate.go

```diff
@@ -127,8 +128,13 @@ var evaluateCmd = &cobra.Command{
 		} else {
 			// get relevant resources from k8s live cluster
 			var err error
+			const ctxTimeoutSeconds = 3
+			ctx := context.Background()
+			ctx, cancel := context.WithTimeout(ctx, ctxTimeoutSeconds*time.Second)
```

Comment:
`ctx` is being overridden.  Can refer to `context.Background` in the call when inializing the actual context